tests/backup/backup_helper.go

```diff
@@ -3936,3 +3937,16 @@ func IsClusterPresent(clusterName string, ctx context.Context, orgID string) (bo
 	return false, nil
 }
 
+// GetConfigObj reads the configuration file and returns a Config object.
+func GetConfigObj() (*backup.Config, error) {
+	var config *backup.Config
+	cm, err := core.Instance().GetConfigMap("cloud-config", "default")
```

Comment:
Can we create a constant string for `cloud-config`

Reply:
added constant